CODING_STYLE.md

```diff
@@ -0,0 +1,254 @@
+0. Formatting
+
+GOLDEN RULE: Follow the style of the existing code when you make changes.
+
+a. Use tabs for leading indentation
+- tab stops are every 4 characters.
+- One indentation level -> exactly one byte (i.e. a tab character) in the source file.
+- If you have run-on lines, indent as you would for a block.
+b. Line widths:
+- Don't worry about having lines of code > 80-char wide.
```

Comment:
Proposed change: Lines should be at most 100 characters wide to make help with diffs and reduce merge conflicts.

Reply:
Perhaps use the same rule as for .sol, 96 or something like that, was merged last week.